api/algolia-sync.js

```
@@ -0,0 +1,161 @@
+
+        const { post } = req.body.current;
+        const event = req.body.event;
+
+        console.log('Event Type:', event);
+        console.log('Post ID:', post.id);
+        console.log('Post Title:', post.title);
+
+        switch (event) {
+            case 'post.published':
+                console.log('Attempting to fetch full post details...');
+                let fullPost;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)